wiki/Beatmap_Editor/Compose/en.md

```diff
@@ -43,70 +43,72 @@ Hold `Alt` to trigger Distance Snap. The value can range from x0.1 - x2.0.
 
-- Triplets*
+- Triplets
 - Has two purple ticks between each white tick
-- Compound timing
+- Compound timing (waltz)
 
 #### Divisor 1/4
 
 ![Divisor 1/4](/wiki/shared/BSD_1_4.jpg "Divisor 1/4")
 
-- Quarter Beats
+- Quarter beats
 - Adds blue ticks, in addition to the red ticks
 - Useful for general mapping.
 
 #### Divisor 1/6
 
 ![Divisor 1/6](/wiki/shared/BSD_1_6.jpg "Divisor 1/6")
 
-- Double Triplets*
+- Double triplets
 - Has two purple ticks between each red tick
-- Compound timing
+- Compound timing (waltz)
 
 #### Divisor 1/8
 
 ![Divisor 1/8](/wiki/shared/BSD_1_8.jpg "Divisor 1/8")
 
-- Eighth Beats
+- Eighth beats
 - Adds yellow ticks, in addition to the red and blue
 
-**Insert Break Time** button will start/stop "Break Time" in current timestamp and **x/y** is current mouse coordinate.
+<!-- No 1/16 divisor? -->
+
+The **Insert Break Time** button places a ![break](/wiki/Glossary/#break) between two well-spaced objects on the timeline.
 
 ### Centre-Left (Selector)
 
 ![Hit Notes Selector](img/BE_HO.jpg "Hit Notes Selector")
 
 **Your construction toolkit, made for DIY (do it yourself).**
 
-**Please note that in osu!standard's default notes hit sounds works differently in osu!mania**
+**Please note that in osu!standard, default note hit sounds work differently than in osu!mania**
 
-- adding clap on a notes will play 'hitclap' **+ hitnormal** sound on hit while in osu!mania: will only play 'hitclap' sound on hit
+- In osu!standard, adding a clap hit sound on a note will play 'hitclap' **+ hitnormal** sound on hit, while in osu!mania, only 'hitclap' will play.
 
 #### Selector
 
 | Buttons / Header (Keyboard Shortcut) | Usage | Description |
 | ------------------------------------ | ----- | ----------- |
-| Sampleset | Auto, Normal, Soft, Drum | This ignore the timing section's Sampleset setting , and add the selected one... e.g. Timing point with **soft 77%**, While this Sampleset set into **Drum** and have **Clap** on the notes, will play default drum-hitnormal 77% + drum-hitclap 77% on hit |
-| Additions | Auto, Normal, Soft, Drum | Additions hitsound means add Clap, Finish, and Whistle with the SELECTED set... e.g. Sampleset **Normal**, Timing point **Soft 77%**, while Additions is set to **Drum** and have **Clap** on the notes, will result of normal-hitnormal 77% + drum-hitclap 77% on hit |
+| Sampleset | Auto, Normal, Soft, Drum | This will ignore the timing section's Sampleset setting and adds the selected sampleset... e.g. Timing point with **soft 77%**, while this Sampleset set into **Drum** and have **Clap** on the notes, will play default drum-hitnormal 77% + drum-hitclap 77% on hit |
+| Additions | Auto, Normal, Soft, Drum | Additions hit sound means add Clap, Finish, and Whistle with the SELECTED set... e.g. Sampleset **Normal**, Timing point **Soft 77%**, while Additions is set to **Drum** and have **Clap** on the notes, will result of normal-hitnormal 77% + drum-hitclap 77% on hit |
```

Comment:
Forces all additions to be taken from a single samplset for one or a number of notes
- only affects clap, whistle, finish hitsounds
- will force them to be taken from either
     - normal
     - soft
     - drum 
     - auto: inherits from the sampleset of the note if it isnt set to auto, if it is set to auto it will inherit from the last green or red timing line